app/trakt_helpers.py

```diff
@@ -14,16 +15,18 @@ def authenticate(authorization=None):
             if Trakt['users/settings']:
                 # Successful authorization
                 return authorization
+    url = Trakt['oauth'].authorize_url('urn:ietf:wg:oauth:2.0:oob')
+    print('| Navigate to: %s' % url)
+    print("| If you get an OAuth error your client_id or client_secret is invalid")
+    webbrowser.open(url, new=2)
```

Comment:
How does this degrade in environments with out a web browser? Seems like a lot of users are using this headless

Reply:
lol thats a great question im not sure but id assumed you would get the credentials right before you turn it on to headless mode

Reply:
we could make it only open the webbrowser if its not headless